src/lib/resolveDefaultsAtRules.js

```diff
@@ -118,22 +118,20 @@ export default function resolveDefaultsAtRules({ tailwindConfig }) {
           }
         }
 
-        if (flagEnabled(tailwindConfig, 'optimizeUniversalDefaults')) {
```

Comment:
This conditional was unnecessary, if you look up at line 94 you can see this code is already inside a block with the exact same conditional.

https://github.com/tailwindlabs/tailwindcss/blob/main/src/lib/resolveDefaultsAtRules.js#L94